projects/observability/src/shared/components/cartesian/chart.ts

```diff
@@ -122,6 +123,11 @@ export interface Axis {
    * Maximum value of the axis. If unset, defaults to maximum value of provided data.
    */
   max?: number;
+
+  /**
+   * Determine the tick count labels (number or time interval)
+   */
+  tickCount?: number | TimeInterval;
```

Comment:
Why would this be a d3 time interval? d3 is not even part of the public api of chart